### ISSUE TITLE
Crash on touch events on IOS since 6.2.0

### ISSUE THREAD

**Issue by sosmo** [openlayers/openlayers#10747]
**Describe the bug**
Panning the map or using gesture zoom on IOS 12 causes OL to crash.

> Cannot read property 'defaultPrevented' of undefined

at MapBrowserEventHandler https://github.com/openlayers/openlayers/blob/88b8b2f7cb95fe6c728767dc5d1194bc04139b60/src/ol/MapBrowserEventHandler.js#L278

The bug was introduced at version 6.2.0. The same doesn't happen on Chrome on PC or Android. It's likely related to commit https://github.com/openlayers/openlayers/commit/88b8b2f7cb95fe6c728767dc5d1194bc04139b60?diff=unified

MapBrowserEventHandler expects "this.originalPointerMoveEvent_" to be set when "handleTouchMove_" gets called, but initially the variable is uninitialized. It should get set at "relayEvent_", but it only gets called after a pointermove event. It seems on IOS Safari "handleTouchMove_" gets called before the listener that fires "relayEvent_", and thus "originalPointerMoveEvent_" is still undefined. "handleTouchMove_" does get called later, it's just the initial call that crashes the app. Thus adding a null check before using "originalPointerMoveEvent_" at this line could be enough to fix the issue

https://github.com/openlayers/openlayers/commit/88b8b2f7cb95fe6c728767dc5d1194bc04139b60?diff=unified#diff-0833750a228e1622e02bcc1779738e7aR278

**Comment by ahocevar** [comment]
@sosmo Thanks for sharing your observation. I don't have an iOS 12 device to test. Can you please create a pull request with your suggested fix?

**Comment by sosmo** [comment]
Thank you for the answer! I tried exploring whether the null check would be enough to cover all bases, but now it seems there's other issues too

Everything works perfectly when I try running the local examples from the repo with `npm start`, even the latest 6.2.2 version. I wasn't even able to reproduce the iOS crash when running from the source js files directly. However, when I tried installing 6.2.1 as an npm module again, it caused a crash on iOS again, and then I noticed that panning the map doesn't work, even on desktop Chromium or Firefox. This didn't happen when I first reported the bug!

Here's a minimal example: https://codesandbox.io/s/loving-resonance-wqcqm [wqcqm.zip](https://github.com/openlayers/openlayers/files/4308074/wqcqm.zip)

When OL is installed through npm, panning doesn't work on the map and on iOS it causes a crash. Changing the OL to version 6.1.1 under "Dependencies" fixes it.

https://github.com/openlayers/openlayers/commit/88b8b2f7cb95fe6c728767dc5d1194bc04139b60 seems to be related, but frankly I have no idea where the problem is. The plain source code works but the transpiled one doesn't.

I'm not sure if this and the iOS crash are related though. I'll submit a pull request anyway, it fixes the iOS crash but not the panning issue.
